tests/framework/utils_cpu_templates.py

```diff
@@ -24,23 +24,23 @@ def get_supported_cpu_templates():
     Return the list of CPU templates supported by the platform.
     """
     # pylint:disable=too-many-return-statements
-    match cpuid_utils.get_cpu_vendor():
-        case cpuid_utils.CpuVendor.INTEL:
-            # T2CL template is only supported on Cascade Lake and newer CPUs.
-            if global_props.cpu_codename == cpuid_utils.CpuModel.INTEL_SKYLAKE:
-                return sorted(set(INTEL_TEMPLATES) - set(["T2CL"]))
+    vendor = get_cpu_vendor()
+    cpu_model = global_props.cpu_model
+    host_linux = global_props.host_linux_version_tpl
+    cpu_codename = global_props.cpu_codename
+
+    match (vendor, cpu_codename, cpu_model):
```

Comment:
No need to wrap it in a tuple, `match vendor, cpu_codename, cpu_model` works fine.

Reply:
Done.